src/interval.jl

```diff
@@ -71,7 +71,27 @@ struct Interval{T} <: AbstractInterval{T}
     end
 end
 
-Interval{T}(f, l, inc::Inclusivity) where T = Interval{T}(convert(T, f), convert(T, l), inc)
+isbounded(a) = !isposinf(a) && !isneginf(a)
+isunbounded(a) = !isbounded(a)
```

Comment:
Should only apply to `AbstractInterval`s

Reply:
Hmm this is actually checking values rather than intervals, as it's mostly just to check if a value is set to ∞ or not. Maybe the name needs to change?

Reply:
My bad then. I think you can leave this as is for now.